src/Router/router.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
```

Comment:
Is there a reason this is a .js file and not .ts?

Reply:
not really

Reply:
it was just generated that way